Detections/SecurityEvent/gte_6_FailedLogons_10m.txt

```diff
@@ -0,0 +1,27 @@
+// Name: Failed Logon Attempts within 10 mins
+//
+// Id: 0777f138-e5d8-4eab-bec1-e11ddfbc2be2
+//
+// Description: A low level alert should occur when failed logon attempts are 6 or higher during a 10 minute period.
+//
+// DataSource: #SecurityEvent
+//
+// Severity: Low
+//
+// QueryFrequency: 10m
+//
+// QueryPeriod: 10m
```

Comment:
similar comment as previous - 20minute QueryPeriod, 10m QueryFrequency?

Reply:
done

Reply:
actually because this query has no join this doesn't work quite the same way. it's now looking back a full 20mins with no condition related to any 10-minute window. there is a concept of time-window join (cf other queries). for now it may be simpler to leave as is though for v1.